src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -495,6 +495,17 @@ private unsafe bool StartWithCreateProcess(ProcessStartInfo startInfo)
                         startupInfo.dwFlags = Interop.Advapi32.StartupInfoOptions.STARTF_USESTDHANDLES;
                     }
 
+                    if (startInfo.WindowStyle != ProcessWindowStyle.Normal)
```

Comment:
Actually, the current behavior looks just like as if we were using ProcessWindowStyle.Normal. So, we can safely map ProcessWindowStyle.Normal => SW_SHOWNORMAL.

Reply:
So specifically this is done to preserve the existing behaviour where `ProcessWindowStyle.Normal` then no `wShowWindow` value is set alongside the `STARTF_USESHOWWINDOW` flag that tells Windows to use the value for `wShowWindow`. I'm not confident that explicitly setting the normal window style is the same as not setting it at all which is what `Normal` currently means so I kept it out. I can change the code so it always just sets this value alongside `STARTF_USESHOWWINDOW` if that's your wish but I was aiming for behaviour that was as close to what it was today.

Reply:
> this is done to preserve the existing behaviour where ProcessWindowStyle.Normal then no wShowWindow value is set alongside the STARTF_USESHOWWINDOW flag.

@iSazonov would your scenario still be covered?

Reply:
PowerShell [always sets the window](https://github.com/PowerShell/PowerShell/blob/66a89826bbaa7bb35c2938c3afc7785c469dbaba/src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs#L2407-L2430) so it would technically be a behaviour change on their end. It really depends on whether dotnet wants to be the one to change the behaviour or let PowerShell do it.

It would make sense for dotnet to align to the same behaviour as `UseShellExecute = true` but I'm just not confident enough to say whether this really matters or not.

Reply:
Let's keep it this way to avoid the breaking change. We can change it later based on user feedback ie: if someone needs this flag to be set for a specific use case.